evm/chain.py

```diff
@@ -249,6 +249,27 @@ def import_block(self, block):
 
         return imported_block
 
+    def ensure_blocks_are_equal(self, block1, block2):
```

Comment:
This seems like it should be a stand-alone function, maybe renamed to `validate_block_equality`.  Doesn't appear to access `self` anywhere in the function body and it should easier to test in isolation that way.